.github/workflows/release.yml

```diff
@@ -0,0 +1,104 @@
+name: Build and upload binaries
+
+on:
+  repository_dispatch:
+    types: [tag-created]
+
+jobs:
+  release:
+    name: Build and release
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            artifact_name: manta-signer
+            asset_name: manta-signer-linux-amd64
+          - os: windows-2019
+            artifact_name: manta-signer
+            asset_name: manta-signer-windows
+          - os: macos-latest
+            artifact_name: manta-signer
+            asset_name: manta-signer-macos-amd64
+    #      - os: macos-10.15
+    #        artifact_name: manta-signer
+    #        asset_name: manta-signer-macos-amd64
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Cache Cargo build dir
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-cargo
+        with:
+          path: ~/.cargo
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('./ui/src-tauri/Cargo.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+      - name: Cache target dir
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-target
+        with:
+          path: ui/src-tauri/target
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('./ui/src-tauri/Cargo.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+      - name: Install Tauri CLI
+        run: cargo install tauri-cli --git 'https://github.com/tauri-apps/tauri' --branch next
```

Comment:
Should we use a more stable version of the CLI or is this stable enough? The documentation recommends some beta version, but maybe that documentation is out-of-date.

Reply:
It looks like tauri is still having issues with clap, the latest version on crates is 1.0.0-beta.7, but that will fail to compiletc